src/Vector4_TEST.cc

```diff
@@ -20,6 +20,7 @@
 #include "ignition/math/Helpers.hh"
 #include "ignition/math/Matrix4.hh"
 #include "ignition/math/Vector4.hh"
+#include "../include/ignition/math/Vector4.hh"
```

Comment:
This doesn't seem necessary, the same header should be included above.

Reply:
Ops, I forgot that rs, will remove